scripts/scripts.js

```diff
@@ -150,7 +150,11 @@ export function splitChildDiv(div, from, to) {
 }
 
 function buildImageCollageForPicture(picture, caption, buildBlockFunction) {
-  const newBlock = buildBlockFunction('image-collage', { elems: [picture, caption] });
+  const captionText = caption.textContent;
```

Comment:
Should you use `innerHTML` here instead of `textContent` to retain any HTML tags inside it.

Reply:
Well I want to get rid of the `<em>` tag so the captions aren't italic anymore so i just need the actual text